### DISCUSSION TITLE
Option for Dry Run that saves all the items that need to be deleted?

### DISCUSSION THREAD

**Discussion by joesmith222** [Klocman/Bulk-Crap-Uninstaller#365] [General]
Hi,

yesterday i needed to uninstall Norton AV or IS program and i used BC Uninstall and of course it begins by using the programs own uninstaller and after that does its work im left with a popup from Norton that says it needs to reboot the computer.  The BC Uninstaller program in unresponsive at this point so im not really left with much choice other than reboot and of course when the computer restarts BC Uninstaller has lost the data afaik and Norton AV cant be found in the list of installed programs to double check if there are any remaining items to be deleted.

Is there a way around this?  I suspect Norton Uninstaller doesnt do as thorough a job as BC would have.

**Comment by Klocman** [comment]
BCU will try to prevent the system from rebooting, but it's best to either tell the uninstaller to not reboot, or if the option is missing terminate the uninstaller from the BCU window.
